README.md

```diff
@@ -1,5 +1,7 @@
 # HTML Project
 
+[LIVE](https://swe2-0-open-8-us.github.io/html-project-NicholasBotorog/)
+
```

Comment:
Great to see that you've included the link to the deployed site.  Suggest that you update the documentation with a brief description of the project, tools/language used, and how to set up.